pyleecan/Methods/Converter/ConvertMC/convert_to_P/convert_slot_to_P.py

```diff
@@ -58,3 +81,26 @@ def convert_slot_to_P(self):
     self.get_logger().info(
         f"Conversion {slot_type} into {type(self.machine.stator.slot).__name__}"
     )
+
+    # selection type of wedge
+    if "Wedge_Model" in self.other_dict["[Winding_Design]"]:
+        wedge = self.other_dict["[Winding_Design]"]["Wedge_Model"]
+
+    else:
+        wedge = "Air"
+
+    # set the correct wedge
+    if wedge == "Air":
+        return 0
+    elif wedge == "Wedge":
+        return 1
+    elif wedge == "Wound_Space":
+        self.get_logger().info(
+            f"Conversion Wound_Space for {type(self.machine.stator.slot).__name__} has not equivalent in pyleecan or has not implement"
+        )
+        self.get_logger().info(f"we define air in place of the wedge")
+        return 0
+    else:
+        self.get_logger().warning(f"Error for conversion of wedge")
+        self.get_logger().info(f"we define air in place of the wedge")
+        return 0
```

Comment:
No return, set wedget_mat to Material or None, then check in other script if a Material is set to add the corresponding rules